### DISCUSSION TITLE
Eclipse Theia v1.52.0

### DISCUSSION THREAD

**Discussion by sgraband** [eclipse-theia/theia#13952] [Release Announcements] [answered]
Hi everyone 👋,

The Eclipse Theia v1.52.0 release is scheduled for later today!
We will inform you once the release starts and completes.

**Comment by sgraband** [comment]
The release will start now, I’ll post an update when it has completed.

**Comment by sgraband** [comment]
The release has completed, thank you to everyone that participated and contributed!

**Comment by JonasHelming** [comment, marked as answer]
[News & Noteworthy](https://eclipsesource.com/blogs/2024/08/06/eclipse-theia-1-52-release-news-and-noteworthy/)
